js/signin.js

```diff
@@ -11,12 +13,33 @@ window.onload = function(){
     const pwEyeIcon = document.querySelector(".fa-solid");
     
     // 로그인 함수
-    function login(){ 
-        if(emailInput.value === userInfo.email && pwInput.value === userInfo.pw){
-            location.href = '/folder.html';
-        }else{
-            emailError.textContent = "이메일을 확인해 주세요.";
-            pwError.textContent = "비밀번호를 확인해 주세요.";
+    async function login(){
+        const userInfo = {
+            email: emailInput.value,
+            password: pwInput.value
+        }
+        try{
+            const response = await fetch(API_PATH_SIGNIN, {
+                method : "POST",
+                headers: {
+                    'Content-Type': 'application/json',
+                },
+                body: JSON.stringify(userInfo)
+            })
+            const result = await response.json();
+            if(response.ok){
+                localStorage.setItem("accessToken", result.data.accessToken);
+                localStorage.setItem("refreshToken", result.data.refreshToken);
+                location.href = "./folder.html";
+            }else{
+                emailError.style.display = "block";
+                pwError.style.display = "block";
+                emailError.textContent = "이메일을 확인해 주세요.";
+                pwError.textContent = "비밀번호를 확인해 주세요.";
+                console.error(result.error);
```

Comment:
```javascript
function setEmailErrorAndStyle(emailError, message, displayStyle) {
    emailError.style.display = displayStyle;
    emailError.textContent = message;
}

```

해당 로직이 많이 반복되어 보이네요. 


```suggestion
      setEmailErrorAndStyle(emailError,"이메일을 확인해 주세요","block");
```

제가 `setEmailErrorAndStyle`이라고 했는데 일반화시켜서 `setErrorAndStyle`이라고 해도 될것 같네요.